src/install/deploySteps.js

```diff
@@ -126,18 +145,17 @@ module.exports = (apps, mode, deployDoc) => {
       })
       .then(deployedDdoc => {
         if (deployedDdoc) {
-          debug('It does, copying config to staged ddoc before writing');
-          primaryDdoc.app_settings = deployedDdoc.app_settings;
+          debug('It does, copying config to settings doc before overwriting');
```

Comment:
As copying config is now not the default I'd re-write these messages so that the default flow doesn't talk about overwriting.

So I'd drop the copying config part of this message and move it into the `updateSettings` code for the situation where we need to do this. I wouldn't even debug out if there are no settings found on the ddoc, as that's what we expect.

Reply:
Fixed.